### PR TITLE
Automation for an identity provider- IBM blue pages LDAP

### PR DESCRIPTION
**Pull request by AdityaHonkalas**
Signed-off-by: Aditya Honkalas <aditya.honkalas@ibm.com>

### CODE REVIEW

File: playbooks/ibmbluepages-ldap-identity-provider.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+- name: Configuring LDAP identity provider with IBM Blue Pages
+  hosts: bastion
+  tasks:
+    - fail:
+        msg: "Please ensure you have set the environment variable IBM_PASSWORD"
```
Comment (yussufsh) [inline playbooks/ibmbluepages-ldap-identity-provider.yml:6]:
```suggestion
        msg: "Please ensure you have set the environment variable IBM_EMAIL_ID and IBM_PASSWORD"
```

File: playbooks/roles/ibmbluepages-ldap-identity-provider/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+|------------------------------|----------|-------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ldap_identity_provider       | yes      | false       | Set to true to enable LDAP test execution                                                                                                                                                                                                                                   |
+| ldap_cr_name                 | no       | empty       | User defined name to the LDAP CR object, if undefined default name will be set as name                                                                                                                                                                                      |
```
Comment (yussufsh) [inline playbooks/roles/ibmbluepages-ldap-identity-provider/README.md:26]: Default should be `ldapidp`

File: playbooks/roles/ibmbluepages-ldap-identity-provider/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+| ldap_user_role               | no       | empty       | User role permission to be granted to the newly created user. the possible role permission can be `"admin", "basic-user", "cluster-admin", "cluster-status", "cluster-reader", "edit", "self-provisioner", "view"`, if empty *cluster-reader* will be assigned by default   |
```
Comment (yussufsh) [inline playbooks/roles/ibmbluepages-ldap-identity-provider/README.md:27]: Default should be `cluster-reader`

**Review by yussufsh — CHANGES_REQUESTED** [review]

**Comment by yussufsh** [comment]
@pravin-dsilva do we still need this?

**Comment by pravin-dsilva** [comment]
> @pravin-dsilva do we still need this?

no, we don't. This can be closed.

**Pull request closed without merging**